js-exercises/cache-function/cacheFunction.js

```diff
@@ -0,0 +1,16 @@
+function cacheFunction(input) {
```

Comment:
I think the parameter name ```inputFn``` would be more clear than using ```input``` from a code readability perspective.

Reply:
changed the variable name